src/qt/modaloverlay.cpp

```diff
@@ -48,6 +53,9 @@ bool ModalOverlay::eventFilter(QObject * obj, QEvent * ev) {
             if (!layerIsVisible)
                 setGeometry(0, height(), width(), height());
 
+            if (animation.state() != QAbstractAnimation::Stopped && animation.endValue().toPoint().y()) {
```

Comment:
nit, `== QPropertyAnimation::Running`? Also add `> 0` or maybe a comment giving some context.

Reply:
According to the [`enum QAbstractAnimation::State`](https://doc.qt.io/qt-5/qabstractanimation.html#State-enum) docs, `state != QAbstractAnimation::Stopped` does not imply `state == QAbstractAnimation::Running` :)

Going to add `>0`.

Reply:
We don't pause right? Oh well, just drop it and always update when > 0 🙈

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/17968#issuecomment-621398261).